environment/docker.go

```diff
@@ -72,9 +72,10 @@ func ConfigureDocker(ctx context.Context) error {
 // Creates a new network on the machine if one does not exist already.
 func createDockerNetwork(ctx context.Context, cli *client.Client) error {
 	nw := config.Get().Docker.Network
-	_, err := cli.NetworkCreate(ctx, nw.Name, types.NetworkCreate{
+	enableIPv6 := true // define a bool variable
```

Comment:
Why make this a variable here? We could also make this a configurable value for wings instead of hard coding it.

Reply:
It now has go be a pointer so that was the easy way. And it always was hardcoded to true there.

Reply:
Just because it always was doesn't mean it can't be now.

Reply:
I added a config toggle for it